cachesetup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# This script sets up the CockroachDB databases and users for the Politeia
+# cache.  This script requires that you have already created all the necessary
+# CockroachDB certificates using the cachecerts.sh script and that you have a
+# CockroachDB instance listening on the default port localhost:26257.
+
+set -ex
+
+# CERTS_DIR must contain client.root.crt, client.root.key, and ca.crt for the
+# CockroachDB instance.
+readonly CERTS_DIR=$1
```

Comment:
Why is this not a sane default instead of required?

Reply:
This depends on the directory that they pass into the `cachecerts.sh` script, which is where the certs get created.

Reply:
I get that but what about using the defaults you document in the README when there is no directory? Maybe @dajohi can weigh in on this.